string supportLeg
float32 duration
